tests/test_integration.py

```diff
@@ -148,10 +148,10 @@ class Class:
     :param z: baz
     """
 
-    def __init__(self, x: bool, y: int, z: Optional[str] = None) -> None:  # noqa: UP007
+    def __init__(self, x: bool, y: int, z: Optional[str] = None) -> None:  # noqa: UP045
```

Comment:
Can we disable `UP045` for the entire tests directory?

Reply:
If you want to do that, you can do

```toml
[tool.ruff.lint.per-file-ignores]
"tests/*" = ["UP45"]
```

Reply:
Could you add that then I'll merge this?

Reply:
Sure, I also added UP006 and UP007 since that’s the same thing